app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -315,6 +373,7 @@ private void hideProgressBar(Achievements achievements) {
             layoutStatistics.setVisibility(View.VISIBLE);
             imageView.setVisibility(View.VISIBLE);
             levelNumber.setVisibility(View.VISIBLE);
+            //levelNumber.setText("You haven't qualified any level yet");
         }
```

Comment:
Also this comment.

Reply:
Fixed int the next commit

Reply:
Also this line: //levelNumber.setText("You haven't qualified any level yet");